### PR TITLE
Tag submodules when we're releasing

### PR DESCRIPTION
**Pull request by SCdF**

### CODE REVIEW

**Comment by SCdF** [comment]
Hey @garethbowen check this out. I have not tested this yet, just thought I'd chuck it up here. Not sure if we want to just test it next time we tag a final, or create some kind of contrivance to test it.

File: scripts/ci/after_success.sh [hunk 1]
```diff
@@ -16,6 +16,16 @@ fi
 cd sentinel && npm install && cd .. && \
 cd api && npm install && cd ..
 
+function tagSubmodules {
+    cd api
+    git tag $TRAVIS_TAG
+    git push --tags
+    cd ../sentinel
+    git tag $TRAVIS_TAG
```
Comment (garethbowen) [inline scripts/ci/after_success.sh]: Can we fix the duplication here? Either pull out another function, `tagSubmodule`, or loop over `['api', 'sentinel']` ?

Comment (SCdF) [inline scripts/ci/after_success.sh]: I mean. We probably can.

Sigh. Fine. I'll learn how stupid @#$@% bash scripting works.

;-)

**Comment by SCdF** [comment]
@garethbowen take a gander We should give it a crack when we crack the next release

**Comment by garethbowen** [comment]
Good job!

**Pull request merged**